react/README.md

```diff
@@ -0,0 +1,105 @@
+# README: Tiny Quickstart (React)
+
+### Overview
+
+This is a minimal app that implements Plaid using a React frontend with an Express/Node backend. After linking a sample bank account, the app retrieves balance information associated with the account and renders it on the home page. For reference, frontend code is in **src/App.jsx**, while backend code is in **server.js**.
+
+If you're looking for a more fully-featured quickstart, covering more API endpoints, available in more languages, and with explanations of the underlying flows, see the official [Plaid Quickstart](https://www.plaid.com/docs/quickstart). 
+
+
+#### Set up your environment
+
+This app uses the latest stable version of Node. At the time of writing, the latest stable version is v16.14.0. It's recommended you use this version of Node to run the app. For information on installing Node, see [How to install Node.js](https://nodejs.dev/learn/how-to-install-nodejs).
+
+#### Install dependencies
+
+Ensure you're in the **react/** folder, then install the necessary dependencies:
+
+```bash
+yarn install
+```
+
+#### Equip the app with credentials
+
+Run the following command to copy the included **.env.example** to a file called **.env**.
+
+```bash
+cp .env.example .env
+```
+
+Fill out the contents of the **.env** file with the [client ID and Sandbox secret in your Plaid dashboard](https://dashboard.plaid.com/team/keys). Don't place quotes (`"`) around the credentials (i.e., `PLAID_CLIENT_ID=adn08a280hqdaj0ad`). Use the "Sandbox" secret when setting the `PLAID_SECRET` variable.
+
+#### Start the server
+
+Start the app by running the following command:
```

Comment:
```suggestion
```
I'm not necessarily going to edit the entire file to conform to this style, but this is an example of how going forward you can be more concise and not spell out things the target audience probably knows.